manifests/charts/istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -88,3 +88,20 @@ data:
 {{- end }}
 ---
 {{- end }}
+
+{{- if .Values.global.multiCluster.clusterName }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: istio-cluster{{- if not (eq .Values.revision "") }}-{{ .Values.revision }}{{- end }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    istio.io/rev: {{ .Values.revision | default "default" }}
+    release: {{ .Release.Name }}
+    istio/multiCluster: true
```

Comment:
Why doesn't this label get applied? The above labels along with some other things that look like they came from operator seem to be there but this gets removed.